src/Dijkstra/AST/Branching.agda

```diff
@@ -76,15 +89,20 @@ module PredTransExtension {O : ASTOps} {T : ASTTypes} (PT : ASTPredTrans O T) wh
   open ASTExtension O
 
   BranchPT : ASTPredTrans BranchOps T
-  ASTPredTrans.returnPT BranchPT  = ASTPredTrans.returnPT  PT
-  ASTPredTrans.bindPT BranchPT    = ASTPredTrans.bindPT    PT
-  ASTPredTrans.opPT BranchPT (Left x) = ASTPredTrans.opPT PT x
-  ASTPredTrans.opPT BranchPT (Right (BCif c)) f P i =
-      (c ≡ true → f (Level.lift true) P i)
-    × (c ≡ false → f (Level.lift false) P i)
-  ASTPredTrans.opPT BranchPT (Right (BCeither e)) f P i =
-      (∀ l → e ≡ Left  l → f (Level.lift (Left l))  P i)
-    × (∀ r → e ≡ Right r → f (Level.lift (Right r)) P i)
+  ASTPredTrans.returnPT BranchPT = ASTPredTrans.returnPT PT
+  ASTPredTrans.bindPT   BranchPT = ASTPredTrans.bindPT   PT
+  ASTPredTrans.opPT     BranchPT (Left x) =
+    ASTPredTrans.opPT PT x
+  ASTPredTrans.opPT     BranchPT (Right (BCif c))     f P i =
+      (       c ≡ true    → f (Level.lift true) P i)
```

Comment:
No big deal, but as long as you are aligning - go all the way and to the `P i` here and below.

Reply:
DONE